docs/modules/databases/mongodb.md

```diff
@@ -3,7 +3,14 @@
 !!! note
     This module is INCUBATING. While it is ready for use and operational in the current version of Testcontainers, it is possible that it may receive breaking changes in the future. See [our contributing guidelines](/contributing/#incubating-modules) for more information on our incubating modules policy.
 
-## Usage example
+The MongoDB module provides two Testcontainers for MongoDB unit testing:
+
+* [MongoDBContainer](#mongodbcontainer) - the core MongoDB database
```

Comment:
```suggestion
* [MongoDBContainer](#mongodbcontainer) - the core MongoDB database with ReplicaSet enabled
```

Reply:
Can we discuss this one? (booked a session next week)
I actually think the current MongoDB test container could use a lot of work (including some updates to its docs). 
I thought about including some of these changes, but perhaps we could do that in a separate PR?

Reply:
> Can we discuss this one?

Sure

> I thought about including some of these changes, but perhaps we could do that in a separate PR?

Yes, let's discuss it first and then a separate PR is the way to go